cves/CVE-2016-2327.yml

```diff
@@ -192,8 +202,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This vulnerability can be exploited with a custom .avi file that would result in the last_frame object having no
+    items in the linesize array, which seems reasonable for a thorough automated tester to be able to discover, as long
+    as the tool is testing .avi files.
```

Comment:
Agree with the Professor here. Having some specifics about certain test cases that might catch this would help create perspective.